dali/python/nvidia/dali/plugin/paddle.py

```diff
@@ -0,0 +1,383 @@
+# Copyright (c) 2017-2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
It would be good to add this iterator to the following tests:
- dali/test/python/test_RN50_data_fw_iterators.py
- dali/test/python/test_fw_iterators.py
- dali/test/python/test_fw_iterators_detection.py

Reply:
Done, all 3 tests passed on my setup.

Reply:
:+1: